src/components/OptionsSelector.js

```diff
@@ -226,54 +326,119 @@ class OptionsSelector extends Component {
     }
 
     render() {
+        const shouldShowFooter = (this.props.shouldShowConfirmButton || this.props.footerContent)
+            && !(this.props.canSelectMultipleOptions && _.isEmpty(this.props.selectedOptions));
+        const defaultConfirmButtonText = _.isUndefined(this.props.confirmButtonText)
+            ? this.props.translate('common.confirm')
+            : this.props.confirmButtonText;
+        const shouldShowDefaultConfirmButton = !this.props.footerContent && defaultConfirmButtonText;
+        const defaultMaxParticipantsReachedMessage = _.isUndefined(this.props.maxParticipantsReachedMessage)
+            ? this.props.translate('common.maxParticipantsReached', {count: this.props.selectedOptions.length})
+            : this.props.maxParticipantsReachedMessage;
+        const shouldShowMaxParticipantsMessage = this.props.maxParticipantsReached && defaultMaxParticipantsReachedMessage;
+        const textInput = (
+            <TextInput
+                ref={el => this.textInput = el}
+                value={this.props.value}
+                label={this.props.textInputLabel}
+                onChangeText={(text) => {
```

Comment:
Hi, we should have used the `onSubmitEditing` prop to enable selection using the software enter key for mobile.

Coming from https://github.com/Expensify/App/issues/21472